src/legacy/core_plugins/kibana/public/management/sections/objects/components/objects_table/objects_table.js

```diff
@@ -677,9 +675,7 @@ class ObjectsTableUI extends Component {
       view: `${type} (${savedObjectCounts[type] || 0})`,
     }));
 
-    const canDeleteSavedObjectTypes = POSSIBLE_TYPES.filter(type => {
-      return this.props.uiCapabilities.savedObjectsManagement[type].delete;
-    });
+    const canDeleteSavedObjectTypes = this.props.uiCapabilities.savedObjectsManagement.delete ? POSSIBLE_TYPES : [];
```

Comment:
Same here, we can simplify this further since we no longer have type-specific checks on this screen.